tests/test_buffer.py

```diff
@@ -824,6 +824,21 @@ def test_SimpleArray_SimpleArrayPlex_type_switch(self):
         self.assertEqual(
             str(type(arrayplex_int32_2)), "<class '_modmesh.SimpleArray'>")
 
+    def test_sort(self):
+        narr = np.random.randint(0, 100, 20, dtype='int32')
```

Comment:
Never use random number in testing.  Test with random number lacks reproducibility unless you are very careful about seeding (but you didn't).

This is a simple algorithm and you can test with carefully crafted fixture.